example.js

```diff
@@ -1,25 +1,25 @@
-const { createQueues, setQueues, UI } = require('./')
+const { setQueues, UI } = require('./')
+const { Queue3 } = require('bullmq/dist/classes/compat')
+const Queue = require('bull')
 const app = require('express')()
 
 const sleep = t => new Promise(resolve => setTimeout(resolve, t * 1000))
 
 const redisOptions = {
-  redis: {
-    port: 6379,
-    host: 'localhost',
-    password: '',
-    tls: false,
-  },
+  port: 6379,
+  host: 'localhost',
+  password: '',
+  tls: false,
 }
 
-const run = () => {
-  setQueues([/* Already defined (bull) queues */]);
-  // Or a single bull queue
-  setQueues(/* Already defined bull queue */);
+const createQueue = name => new Queue(name, { redis: redisOptions })
+const createQueue4 = name => new Queue3(name, { connection: redisOptions })
```

Comment:
this naming is causing me some confusion 😅 queue3 is the version 3 coming from bullmq, which is version 4?
maybe we can use names such as Queue3 and QueueMQ? it's a bit odd but it aligns with the lib name, what are your thoughts?

Reply:
Yep, make sense! I will update it now